deepchem/models/torch_models/layers.py

```diff
@@ -0,0 +1,36 @@
+import math
+try:
+  import torch
+  import torch.nn as nn
+except:
+  raise ImportError('These classes require Torch to be installed.')
+
+
+class ScaleNorm(nn.Module):
+  """Apply Scale Normalization to input.
+
+    All G values are initialized to sqrt(d).
+
+    References
+    ----------
+    .. [1] Lukasz Maziarka et al. "Molecule Attention Transformer" Graph Representation Learning workshop and Machine Learning and the Physical Sciences workshop at NeurIPS 2019. 2020. https://arxiv.org/abs/2002.08264
+    """
+
+  def __init__(self, scale, eps=1e-5):
+    """Initialize a ScaleNorm layer.
+
+        Parameters
```

Comment:
Formatting here is off. Check that you have the right version of `yapf` (0.22). You might need to manually fix

Reply:
I am using `yapf 0.22`. I've fixed the docstrings manually.